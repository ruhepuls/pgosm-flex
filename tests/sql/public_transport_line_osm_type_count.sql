SELECT osm_type COLLATE "C", osm_subtype COLLATE "C", COUNT(*)
    FROM osm.public_transport_line
    GROUP BY osm_type COLLATE "C", osm_subtype COLLATE "C"
    ORDER BY osm_type COLLATE "C", osm_subtype COLLATE "C"
;