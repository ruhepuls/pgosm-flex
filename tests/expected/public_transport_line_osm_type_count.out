aerialway|animal_crossing|1
railway|abandoned|31
railway|disused|22
railway|facility|18
railway|light_rail|2
railway|rail|512
railway|subway|225
railway|tram|37
